assets/js/results-list.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+
+export function ResultsTitle(props){
+    return(
+      <p className="results-title">Results for {props.city}</p>
+    );
+}
+
+export function SingleResult(props){
```

Comment:
usually I see this as export const SingleResult = (props) => {}

Reply:
would that mean that I have to change how the render() function is written as well? To be consistent with the version of javascript that I am using?

Reply:
no, you'd just change the definition of the component